utils/backup/S3/fo-backup-s3

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# Copyright (C) Siemens AG 2017
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# version 2 as published by the Free Software Foundation.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+if [[ $(id -u) -ne 0 ]]; then
+ echo "Please run as root"
+ exit 1
+fi
+
+if [[ ${1} = "-l" ]]; then
+ LOGGING=${2}
+ if [ -z ${LOGGING} ]; then
```

Comment:
it would be better to write `[ -z "${LOGGING}" ];` which is best practice (AFAIK).